### ISSUE TITLE
VPAT — Keyboard user cannot navigate to file submission box

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#3155]
Tab key cannot gain focus to the file submission box that is rendered for gradeable submissions.  This prevents keyboard-only users from submitting any files for gradeables.

Submission box rendering code:
https://github.com/Submitty/Submitty/blob/c99b355092dd43cfae0babee2fce28e9c54d744b/site/app/templates/submission/homework/SubmitBox.twig#L129-L142
